web/templatetags/membership_tags.py

```
@@ -0,0 +1,27 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def format_price(value, period=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `format_price`

Add return type annotation: `str`

(ANN201)